packages/legend-query/src/stores/postFilterOperators/QueryBuilderPostFilterOperatorHelper.ts

```diff
@@ -66,6 +66,7 @@ export const getColumnMultiplicity = (
 export const buildPostFilterConditionExpression = (
   filterConditionState: PostFilterConditionState,
   operator: QueryBuilderPostFilterOperator,
+  operatorFunctionFullPath?: string,
```

Comment:
--> `operatorFunctionFullPath: string | undefined,`

Let's document this flag as well.

```ts
/**
 * If provided, this will be used to construct the simple
 * function expression for the function with the specified
 * name. If not provided, we will fall back to use the TDS column getter function expression. This is the case because with TDS, we are provided some filter-like operators, e.g. IS_NULL, IS_NOT_NULL, etc.
 */
```